src/backend/gl/src/queue.rs

```diff
@@ -256,7 +256,9 @@ impl CommandQueue {
                 self.state.index_buffer = Some(buffer);
                 unsafe { gl.BindBuffer(gl::ELEMENT_ARRAY_BUFFER, buffer) };
             }
-//          com::Command::BindVertexBuffers(_data_ptr) =>
+            com::Command::BindVertexBuffer(buffer) => unsafe {
```

Comment:
I think we should roughly model the internal commands according the GL API:
```rust
BindBuffer => glBindBuffer
BindVertexBuffer => glBindVertexBuffer (4.3+, ES 3.1+)
BindVertexBuffers => glBindVertexBuffers (4.4+)
```
I guess it's easier to reason about the internals and moves computation from submit to recording (checking capabilities).